### PR TITLE
Cleanup the getting started guide

### PR DESCRIPTION
**Pull request by nfnt**
**What this PR does / why we need it**:
Removes details that are already covered elsewhere. Adds links to sections that provide details.

### CODE REVIEW

File: content/docs/developing-operators/getting-started.md [hunk 1]
```diff
@@ -30,15 +30,7 @@ This is a pretty normal Kubernetes YAML file defining a deployment. However, you
...
-
-**Note:** If you want to install the result of the following steps with doing them manually, you can clone the KUDO repository and run the example from there:
-
-```bash
-git clone https://github.com/kudobuilder/operators.git
-cd operators
-kubectl kudo install ./repository/first-operator/operator/
-```
+Now your first operator is ready and you can install it to your cluster. You can do this by invoking `kubectl kudo install ./first-operator` where `./first-operator` is a relative path to the folder containing your operator. To do this, you need to have the KUDO CLI installed - [follow the instructions here](cli.md), if you haven't already. Various resources will be installed for your operator, among them `Operator`, `OperatorVersion` and `Instance` as described in [What is KUDO](TODO).
```
Comment (zen-dog) [inline content/docs/developing-operators/getting-started.md]:
```suggestion
Now your first operator is ready and you can install it to your cluster. You can do this by invoking `kubectl kudo install ./first-operator` where `./first-operator` is a relative path to the folder containing your operator. To do this, you need to have the KUDO CLI installed - [follow the instructions here](cli.md), if you haven't already. Various resources will be installed for your operator, among them `Operator`, `OperatorVersion` and `Instance`, as described in, [What is KUDO](what-is-kudo.md).
```

File: content/docs/developing-operators/getting-started.md [hunk 2]
```diff
@@ -30,15 +30,7 @@ This is a pretty normal Kubernetes YAML file defining a deployment. However, you
 
 <<< @/kudo/test/integration/first-operator-test/first-operator/params.yaml
```
Comment (zen-dog) [inline content/docs/developing-operators/getting-started.md:43]: I think this section needs some more love. I agree that we should 't repeat ourselves, however:
- we jump directly into the yaml files, without even explaining *what operator we're building* and what should be expected at the end
- the _"Details for the concepts"_ part is important but should probably be placed at the beginning as a "Prerequisite" to the rest of it

**Review by zen-dog — COMMENTED** [review]
```
  _______________________________
< Need a bit more love, this one. >
  -------------------------------
         \   ^__^ 
          \  (oo)\_______
             (__)\       )\/\
                 ||----w |
                 ||     ||
```

**Comment by zen-dog** [comment]
Ok, so I reworked it a bit more. Wdyt, @nfnt ?

**Review by zen-dog — APPROVED** [review]

File: content/docs/developing-operators/getting-started.md [hunk 3]
```diff
@@ -1,22 +1,34 @@
...
+
+* [Operator packages](developing-operators/packages.md)
+* [Plans](developing-operators/plans.md)
+* [Tasks](developing-operators/tasks.md)
+* [Parameters](developing-operators/parameters.md)
+* [Templates](developing-operators/templates.md)
+
+## Package Structure
+
+Out first operator will deploy nginx to the cluster. If you're thinkg that one doesn't need a KUDO operator to do this, you're absolutely right. It hardly showcases KUDOs stong sides but this is a `hello-world` type of operator and we have to start somewhere.
```
Comment (zen-dog) [inline content/docs/developing-operators/getting-started.md]:
```suggestion
Our first operator will deploy Nginx to the cluster. If you're thinking that one doesn't need a KUDO operator to do this, you're absolutely right. It hardly showcases KUDOs strong sides but this is a `hello-world` type of operator and we have to start somewhere.
```

File: content/docs/developing-operators/getting-started.md [hunk 4]
```diff
@@ -1,22 +1,34 @@
 # Getting Started
 
-In this section we’ll start by developing your first operator and we’ll follow up with in-depth explanation of the underlying concepts.
+## Prerequisites
+
+It's time to create your first operator. But before you proceed make sure you're up-to-date on all the important KUDO concepts:
```
Comment (zen-dog) [inline content/docs/developing-operators/getting-started.md]:
```suggestion
It's time to create your first operator. But before you proceed, make sure you're up-to-date on all the important KUDO concepts:
```

File: content/docs/developing-operators/getting-started.md [hunk 5]
```diff
@@ -30,15 +42,7 @@ This is a pretty normal Kubernetes YAML file defining a deployment. However, you
...
-
-**Note:** If you want to install the result of the following steps with doing them manually, you can clone the KUDO repository and run the example from there:
-
-```bash
-git clone https://github.com/kudobuilder/operators.git
-cd operators
-kubectl kudo install ./repository/first-operator/operator/
-```
+Now your first operator is ready and you can install it to your cluster. You can do this by invoking `kubectl kudo install ./first-operator` where `./first-operator` is a relative path to the folder containing your operator. To do this, you need to have the KUDO CLI installed - [follow the instructions here](cli.md), if you haven't already. Various resources will be installed for your operator, among them `Operator`, `OperatorVersion` and `Instance` as described in [What is KUDO](TODO).
```
Comment (zen-dog) [inline content/docs/developing-operators/getting-started.md]:
```suggestion
Now your first operator is ready and you can install it to your cluster. You can do this by invoking `kubectl kudo install ./first-operator` where `./first-operator` is a relative path to the folder containing your operator. To do this, you need to have the KUDO CLI installed - [follow the instructions here](cli.md), if you haven't already. Various resources will be installed for your operator, among them `Operator`, `OperatorVersion` and `Instance`, as described in, [What is KUDO](TODO).
```

File: content/docs/developing-operators/getting-started.md [hunk 6]
```diff
@@ -30,15 +42,7 @@ This is a pretty normal Kubernetes YAML file defining a deployment. However, you
...
-
-**Note:** If you want to install the result of the following steps with doing them manually, you can clone the KUDO repository and run the example from there:
-
-```bash
-git clone https://github.com/kudobuilder/operators.git
-cd operators
-kubectl kudo install ./repository/first-operator/operator/
-```
+Now your first operator is ready and you can install it to your cluster. You can do this by invoking `kubectl kudo install ./first-operator` where `./first-operator` is a relative path to the folder containing your operator. To do this, you need to have the KUDO CLI installed - [follow the instructions here](cli.md), if you haven't already. Various resources will be installed for your operator, among them `Operator`, `OperatorVersion` and `Instance`, as described in, [What is KUDO](what-is-kudo.md).
```
Comment (nfnt) [inline content/docs/developing-operators/getting-started.md]:
```suggestion
Now your first operator is ready and you can install it to your cluster. You can do this by invoking `kubectl kudo install ./first-operator` where `./first-operator` is a relative path to the folder containing your operator. To do this, you need to have the KUDO CLI installed - [follow the instructions here](cli.md), if you haven't already. Various resources will be installed for your operator, among them `Operator`, `OperatorVersion` and `Instance`, as described in, [What is KUDO](../what-is-kudo.md).
```

File: content/docs/developing-operators/getting-started.md [hunk 7]
```diff
@@ -1,22 +1,34 @@
 # Getting Started
 
-In this section we’ll start by developing your first operator and we’ll follow up with in-depth explanation of the underlying concepts.
+## Prerequisites
+
+It's time to create your first operator. But before you proceed, make sure you're up-to-date on all the important KUDO concepts:
+
+* [Operator packages](developing-operators/packages.md)
```
Comment (nfnt) [inline content/docs/developing-operators/getting-started.md]:
```suggestion
* [Operator packages](packages.md)
```
Links are relative to the current file. Here and below.

**Comment by nfnt** [comment]
@zen-dog Thank you for the improvements, they're great!

**Pull request merged**